Library/Formula/ekg2.rb

```diff
@@ -1,28 +1,115 @@
 class Ekg2 < Formula
   desc "Multiplatform, multiprotocol, plugin-based instant messenger"
-  homepage "http://ekg2.org"
+  homepage "http://en.ekg2.org"
   url "http://pl.ekg2.org/ekg2-0.3.1.tar.gz"
   sha256 "6ad360f8ca788d4f5baff226200f56922031ceda1ce0814e650fa4d877099c63"
+  revision 1
 
   depends_on "pkg-config" => :build
   depends_on "readline"
   depends_on "libgadu" => :optional
+  depends_on "openssl"
+
+  # Fix the build on OS X 10.9+
+  # http://bugs.ekg2.org/issues/152
+  patch :DATA
```

Comment:
Forgot my password for their issue tracker; will add a new gist once I get that sorted.